napari/layers/_shapes_layer/shapes/rectangle.py

```diff
@@ -9,8 +9,10 @@ class Rectangle(Shape):
 
     Parameters
     ----------
-    data : np.ndarray
-        Nx2 array of vertices specifying the shape.
+    data : (4, 2) array
+        Either a (2, 2) array specifying the two corners of an axis aligned
```

Comment:
if you can have two different shapes, you should specify two different shapes in the type

Reply:
done